maintenance/wikia/sync_user_groups.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Wikia\DependencyInjection\Injector;
+use Wikia\UserGroups\Maintenance\UserGroupUpdater;
+
+$wgCommandLineSilentMode = true;
+
+require_once(dirname(__FILE__).'/../Maintenance.php');
+
+class SyncWikiUserGroups extends Maintenance {
+
+	public function __construct() {
+		parent::__construct();
+
+		$this->addOption('wiki_id', '', true, true);
+	}
+
+	public function loadParamsAndArgs($self = null, $opts = null, $args = null) {
+		parent::loadParamsAndArgs($self, $opts, $args);
+
+		if (!empty($this->mOptions['wiki_id'])) {
```

Comment:
Can we have this the other way around, so the script is usable with `run_maintenance`?